### PR TITLE
FIX QR code generator if invoice with bank_account and bank_code sepa…

### PR DESCRIPTION
**Pull request by truhlik**
FIX for #33

### CODE REVIEW

**Comment by timthelion** [comment]
While this may work in some bank apps it doesn't seem to follow [the standard](https://cbaonline.cz/upload/538-qr-kody-2015.pdf) ( Page 7: ACC)

qr-acc

Instead we need to know the IBAN of the account and perhaps the BIC or SWIFT code as well (Though that seems to be optional).

**Comment by timthelion** [comment]
It looks like it is possible to generate an IBAN for our purposes with the info we have https://en.wikipedia.org/wiki/International_Bank_Account_Number#Generating_IBAN_check_digits
But I'm quite sure that it is easier and less error prone just to add a new IBAN field to the Address object https://github.com/by-cx/InvoiceGenerator/blob/cba6358cb9af2c777455f91526e607bb659e4220/InvoiceGenerator/api.py#L22

**Comment by timthelion** [comment]
Aha, sorry, I didn't realize that the QRPlatba package accounts for this in their API https://pypi.org/project/qrplatba/ this pr looks good to me.

**Comment by timthelion** [comment]
Here is the checksum code that I didn't feel like implementing https://github.com/ViktorStiskala/python-qrplatba/blob/096af38dcf42ab2a46cc82e057cb3d58caa663b2/qrplatba/spayd.py#L56

File: InvoiceGenerator/api.py [hunk 1]
```diff
@@ -356,7 +356,7 @@ def _fill(self, invoice):
         from qrplatba import QRPlatbaGenerator
 
         qr_kwargs = {
-            'account': invoice.provider.bank_account,
+            'account': invoice.provider.bank_account_str,
```
Comment (timthelion) [inline InvoiceGenerator/api.py:359]: Zapoměl jsi () ;)

**Comment by timthelion** [comment]
I fixed this and the tests and added missing parentheses here: https://github.com/by-cx/InvoiceGenerator/pull/42

**Comment by PetrDlouhy** [comment]
I merged #42, so I am closing this.

**Pull request closed without merging**